projects/azure/README.md

```diff
@@ -22,3 +22,43 @@ Run job:
 ```
 ci-operator --config ci-operator/config/openshift/openshift-azure/master.json --namespace=namespace-name --git-ref=openshift/openshift-azure@master --template ci-operator/templates/cluster-launch-e2e-azure.yaml --secret-dir $(pwd)/cluster/test-deploy/azure/
 ```
+
+# Secret rotation
+
+OSA jobs are using `Web API App` credentials on Azure to run jobs. If for some reason you need to rotate secret, follow this process:
```

Comment:
It would be clearer if you described up front that there are 2 secrets in 2 namespaces that slightly differ in syntax and what they are used for.
Can you open a follow-up issue to cover converging the 2 secret syntaxes into one.  I prefer the azure namespace style but fundamentally I don't care too much

Reply:
@mjudeikis ^